source/triggers/aws-eventbridge.txt

```diff
@@ -152,13 +170,358 @@ Procedure
       rules that trigger on events from that partner event source. For more
       information, see :aws-docs:`Creating a Rule That Triggers on a SaaS Partner Event </eventbridge/latest/userguide/create-rule-partner-events.html>`.
 
+.. _eventbridge-error-handling:
+
+Custom Error Handling
+---------------------
+
+.. note:: Only Database Triggers Support Custom Error Handlers
+
+   Currently, only database triggers support custom error handling. 
+   Authentication triggers and scheduled triggers do not support 
+   custom error handling at this time. 
+
+.. figure:: /images/eventbridge-error-handling-diagram.png
+   :alt: EventBridge Custom Error Handling Flow Diagram.
+
+You can create an error handler to be executed on a trigger failure, 
+when retry does not succeed. Custom error handling allows you to determine
+whether an error from AWS EventBridge is critical enough to suspend the Trigger,
+or if it is acceptable to ignore the error and continue processing other events.
+For more information on suspended databse triggers, refer to 
+:ref:`Suspended Triggers <suspended_triggers>`.
+
+Create a New Custom Error Handler
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. tabs-realm-admin-interfaces::
+
+   .. tab::
+      :tabid: ui
+
+      .. figure:: /images/eventbridge-custom-function.png
+         :alt: The EventBridge custom error handling configuration in the UI.
+
+      .. procedure::
+
+         .. step:: Create a New Error Handler
+
+            In the :guilabel:`Configure Error Function` section, select 
+            :guilabel:`+ New Function`.
+
+            You can also select an existing Function, if one is already defined,  
+            from the dropdown. For more information, see 
+            :ref:`Define a Function <define-a-function>`.
+
+         .. step:: Name the New Function
+
+            Enter a unique, identifying name for the function in the :guilabel:`Name` field. 
+            This name must be distinct from all other functions in the application.
+
+         .. step:: Write the Function Code
+
+            In the :guilabel:`Function` section, write the function in the function editor.
```

Comment:
suggested revision to avoid the repetition and provide slightly more info
```suggestion
            In the :guilabel:`Function` section, write the JavaScript code directly in the function editor. The function editor contains a default function that you can edit as needed. For more information on creating functions, refer to the :ref:`Functions <...>` documentation.
```